tests/output.rs

```diff
@@ -44,62 +44,106 @@ struct ExpectedMatch {
     pub range: Range,
 }
 
-impl ExpectedMatch {
-    pub fn expected_output_text(&self) -> String {
```

Comment:
I moved this onto a method on `OutputMode` below (which seems questionable given the number of references to fields on this struct)